demisto_sdk/tests/test_files/content_slim/Packs/Sample02/IndicatorFields/incidentfield-sample2.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
any reason this is a full file?

Reply:
Yes, Findtype knows how to detect it only by the file content, due to the fact that it starts with the incident field.